plugin/session_view.py

```diff
@@ -80,6 +80,7 @@ def on_before_remove(self) -> None:
             self.view.erase_regions(self.diagnostics_key(severity, False))
             self.view.erase_regions(self.diagnostics_key(severity, True))
         self.view.erase_regions("lsp_document_link")
+        self.session_buffer.remove_all_inlay_hints()
```

Comment:
Still not right. This can be moved back to previous place but you should only call it there when `self.session_views` becomes empty.

Reply:
[3823e3b](https://github.com/sublimelsp/LSP/pull/2018/commits/3823e3b7c69b2cf9d7ec29321f38d0fe426d49a7) +
[2523ed8](https://github.com/sublimelsp/LSP/pull/2018/commits/2523ed8743e7184a93ec1e511fffc65475acbca3) ?